plugins/services/src/js/components/modals/ServiceSuspendModal.js

```diff
@@ -80,34 +80,46 @@ class ServiceSuspendModal extends React.Component {
     );
   }
 
-  render() {
-    const { isPending, onClose, open, service, suspendItem } = this.props;
-
-    let itemText = "Service";
-    let serviceName = "";
+  getServiceLabel() {
+    const { service } = this.props;
 
     if (service instanceof Pod) {
-      itemText = "Pod";
+      return "Pod";
     }
 
     if (service instanceof ServiceTree) {
-      itemText = "Group";
+      return "Group";
     }
 
-    if (service) {
-      serviceName = service.getId();
-    }
+    return "Service";
+  }
+
+  getModalHeading() {
+    const serviceLabel = this.getServiceLabel();
 
-    const heading = (
+    return (
       <ModalHeading>
-        Suspend {itemText}
+        Suspend {serviceLabel}
       </ModalHeading>
     );
+  }
+
+  render() {
+    const { isPending, onClose, open, service, suspendItem } = this.props;
+
+    let itemText = "";
+    let serviceName = "";
+    const serviceLabel = this.getServiceLabel();
+    itemText += ` ${serviceLabel}`;
```

Comment:
itemText looks like it is the same with serviceLabel and doesn't change. We could probably remove it altoghere and use only the serviceLabel